scripts/add_existing_baseyear.py

```diff
@@ -434,12 +443,24 @@ def add_heating_capacities_installed_before_baseyear(
         else:
             efficiency = costs.at[costs_name, "efficiency"]
 
-        for i, grouping_year in enumerate(grouping_years):
-            if int(grouping_year) + default_lifetime <= int(baseyear):
-                continue
+        valid_grouping_years = pd.Series(
+            [
+                int(grouping_year)
+                for grouping_year in grouping_years
+                if int(grouping_year) + default_lifetime > int(baseyear)
```

Comment:
this should be >= imo

Reply:
to be consistent with previous definition, I think it shoud not.

Let's say `grouping_year=2000`, `baseyear=2020`, `default_lifetime=20`
Then before
`int(grouping_year) + default_lifetime <= int(baseyear)`
 2020 + 20 <= 2020 -> True -> not considered
Now with your sign ">="
2020 + 20 >= 2020 -> True -> considered as valid grouping year

So it should be `>`. This is also consistent with our definition that operation of an asset starts in 2020, with a lifetime of 20 years it should end operation then add the end of 2019.